mscore/musescore.cpp

```diff
@@ -1202,6 +1202,26 @@ MuseScore::MuseScore()
 
       populateNoteInputMenu();
 
+      //-------------------------------
+      //    Feedback Tool Bar
+      //-------------------------------
+
+      feedbackTools = new QToolBar("", this);
+      feedbackTools->setObjectName("feedback-tools");
+      // Add the toolbar to the bottom and forbid to move it...
+      feedbackTools->setMovable(false);
+      feedbackTools->setFloatable(false);
+      addToolBar(Qt::BottomToolBarArea, feedbackTools);
```

Comment:
I guess changing this to `Qt::TopToolBarArea` would move this up to where all the other toolbars are, right?

Reply:
Or just removing this and change `feedbackTools = new QToolBar("", this);` above to `feedbackTools = addToolBar("");`